functions/reportToIWF.ts

```diff
@@ -72,11 +73,13 @@ export const handler: ServerlessFunctionSignature<EnvVars, Event> = TokenValidat
         ? parseInt(context.IWF_API_COUNTRY_CODE, 10)
         : null;
 
+      const channelID = context.IWF_API_CHANNEL_ID ? parseInt(context.IWF_API_CHANNEL_ID, 10) : 51;
```

Comment:
Is it fine that we still default to `51` if there's no custom id? Or is that a required field from now on? Just asking in case the requirements of IWF changed.

Reply:
From what I read in the ticket, they were specifically asking for those 3 accounts to be changed - I'm going to assume 51 is still a valid default until told otherwise